src/main/java/org/jabref/gui/desktop/os/Linux.java

```diff
@@ -171,9 +172,34 @@ public Path getApplicationDirectory() {
 
     @Override
     public Path getDefaultFileChooserDirectory() {
-        return Path.of(Objects.requireNonNullElse(
-                System.getenv("XDG_DOCUMENTS_DIR"),
-                System.getProperty("user.home") + "/Documents")
-        );
+        String xdgDocumentsDir = System.getenv("XDG_DOCUMENTS_DIR");
+        if (xdgDocumentsDir != null) {
+            return Path.of(xdgDocumentsDir);
+        }
+
+        // Make use of xdg-user-dirs
+        // See https://www.freedesktop.org/wiki/Software/xdg-user-dirs/ for details
+        try {
+            Process process = new ProcessBuilder("xdg-user-dirs", "DOCUMENTS").start();
```

Comment:
the script is called `xdg-user-dir` without the s. Its a mess i know. The ""Toolkit"" is called `-dirs` but the script that you actually run is called `-dir`

Reply:
I tested it anyways by coping the script with the modified name into my path, and it successfully puts `/home/<USER>/Dokumente` into the "Main file directory" box.